CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 ## Changelog
 
+### 2.1.0 (2020-11-11)
```

Comment:
I think it's useful to add a SDKCF-1234 text (not link) at the end if there is a relevant JIRA